.toys/ci.rb

```diff
@@ -288,7 +295,7 @@ def filter_gem_dirs dirs
     else
       false
     end
-  end.sort
+  end.uniq.sort
```

Comment:
The `uniq` shouldn't be necessary. The input `dirs` should already be uniquified because it is populated from a set.

Reply:
:+1: